### PR TITLE
Drop support for Python 3.6 and 3.7

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #8855

### CODE REVIEW

File: .ci/ansible/Containerfile.j2 [hunk 1]
```diff
@@ -10,7 +10,7 @@ ADD {{ item.source }} {{ item.source }}
 # Install python packages
 # Hacking botocore (https://github.com/boto/botocore/pull/1990)
 
-RUN pip3 install \
+RUN pip3 install packaging \
```
Comment (daviddavis) [inline .ci/ansible/Containerfile.j2:13]: Should be fixed by https://github.com/pulp/pulp-oci-images/pull/104

**Review by fao89 — APPROVED** [review]

**Review by mdellweg — APPROVED** [review]
This is applying the plugin_template, right?

**Comment by daviddavis** [comment]
The plugin_template changes are here: https://github.com/pulp/plugin_template/pull/436

But technically I didn't apply the plugin_template. I wrote a script to update each repo as it was more convenient and also I didn't want to update the entire CIs for each plugin since I wasn't sure what state they were in.

**Comment by daviddavis** [comment]
I'm happy to drop the CI changes from this PR and open a separate PR where I run the plugin_template to update all the CI files.

**Review by bmbouter — APPROVED** [review]
This looks good to me. Thanks @daviddavis !

**Pull request merged**